blaze/expr/arithmetic.py

```diff
@@ -220,7 +221,7 @@ def scalar_coerce(_, val):
 
 @dispatch(ct.DateTime, _strtypes)
 def scalar_coerce(_, val):
-    return dt_parse(val)
+    return pd.Timestamp(val)
```

Comment:
Hrm, that's unfortunate.  I'm surprised that sqlalchemy doesn't manage python datetimes well.